src/libdredd/src/mutate_ast_consumer.cc

```diff
@@ -284,7 +285,7 @@ std::string MutateAstConsumer::GetRegularDreddPreludeC(
   result << "#endif\n";
   result << "\n";
   result << "static thread_local int __dredd_some_mutation_enabled = 1;\n";
-  result << "static int __dredd_enabled_mutation(int local_mutation_id) {\n";
+  result << "static bool __dredd_enabled_mutation(int local_mutation_id) {\n";
```

Comment:
I'm fine with using `bool` here as it is more readable, but because C does not have a `bool` type, I think this will end up being the same as `int`. Therefore this change might not have any impact. Again - I am fine with the change, but wanted to bring this up in case your reasoning was that it would have impact.